tests/cli/test_felix.py

```diff
@@ -99,6 +99,46 @@ def request_felix_landing_page(_result):
         response = test_client.post('/register', data={'address': felix.blockchain.interface.w3.eth.accounts[-1]})
         assert response.status_code == 200
 
+        # register a recipient by posting json
+        response = test_client.post(
+            '/register',
+            data=json.dumps({'address': felix.blockchain.interface.w3.eth.accounts[-2]}),
+            content_type='application/json'
+        )
+        assert response.status_code == 200
+
+        # same address again should fail with 400
+        response = test_client.post(
+            '/register',
+            data=json.dumps({'address': felix.blockchain.interface.w3.eth.accounts[-2]}),
+            content_type='application/json'
+        )
+        assert response.status_code == 400
```

Comment:
You can also check that the response data is correct here.

For example: `assert response.json() == 'already registered'`